BUILDING_ON_WINDOWS.md

```diff
@@ -37,6 +38,16 @@ Notes:
 Note: This installation will take about 2 GB of disk space.
 
 Once Qt is done installing, make sure you add its bin directory to your `PATH` (e.g. `C:\Qt\5.15.2\msvc2019_64\bin`)
+<details>
+   <summary>How to add Qt to PATH</summary>
+   
+1. Type "path" in the Windows start menu and click `Edit the system environment variables`.
+2. Click the `Environment Variables...` button bottom right.
+3. In the `System variables` section, scroll down until you find `Path` and double click it.
+4. Click the 'New' button top right and paste in the file path for your Qt installation (e.g. `C:\Qt\5.15.2\msvc2019_64\bin` by default).
```

Comment:
Should probably be whatever Qt 6 path is relevant now

Reply:
Ok, I changed the other Qt paths too. You can unresolve this if there is still an issue.